packages/pds/src/sequencer/events.ts

```diff
@@ -128,6 +128,20 @@ export const formatSeqHandleUpdate = async (
   }
 }
 
+export const formatSeqTombstone = async (
```

Comment:
one thing is that these should invalidate all prior events from the user

take a look at `sequenceEvt` where "rebases" invalidate prior "rebases" & "appends" and "handles" invalidate prior "handles" 

(also merge in main first cause we just made some some changes to invalidation)